ui/src/app/core/manage-plugins/custom-plugins/homebridge-honeywell-home/homebridge-honeywell-home.component.ts

```diff
@@ -50,10 +50,9 @@ export class HomebridgeHoneywellHomeComponent implements OnInit, OnDestroy {
     if (!this.pluginConfig) {
       this.pluginConfig = {
         platform: this.schema.pluginAlias,
-        name: 'HoneywellHome',
+        name: this.schema.pluginAlias,
         options: {
-          ttl: 60,
-          verbose: false,
+          ttl: this.schema.pluginAlias,
```

Comment:
I don't think this is what you want here?

Reply:
should we just leave that blank? and remove the options: { } object?

Reply:
done